src/session/sessionReducer.js

```diff
@@ -145,9 +148,16 @@ export default (state: SessionState = initialState, action: Action): SessionStat
         isActive: action.isActive,
       };
 
+    case INITIAL_FETCH_START:
+      return {
+        ...state,
+        loading: true,
+      };
+
     case INITIAL_FETCH_COMPLETE:
       return {
         ...state,
+        loading: false,
```

Comment:
This commit is WIP, because not sure why we don't have session reducer test for this case. Should I add it?

Reply:
Sure, why not.

Reply:
Hey, I have added few tests. Can you please review and let me know, whether I am going in the right direction.